### PR TITLE
History page

### PR DESCRIPTION
**Pull request by VedanthB**
- added history feat
- refactored watch later, likes, playlist  pages

### CODE REVIEW

File: src/frontend/context/constants/likeVideosConstants.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const likeVideosActions = {
```
Comment (Mohitkumar33) [inline src/frontend/context/constants/likeVideosConstants.js:1]: Nice variable naming

**Review by Mohitkumar33 — APPROVED** [review]
Good to merge

File: src/frontend/components/history-page/HistoryPageCard.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+              className="btn btn-link-amber"
+            >
+              {" "}
+              remove history
+            </button>
+            {/* <button className="btn btn-solid-amber shadow-lg text-white">
+              {" "}
+              remove watch later
+            </button> */}
+
```
Comment (Hardiegogo) [inline src/frontend/components/history-page/HistoryPageCard.js:96]: Remove unused code

**Review by Hardiegogo — APPROVED** [review]
Nicely structured code,
I liked how you have abstracted out action variables.

**Pull request merged**